core/src/main/java/jenkins/slaves/JnlpSlaveAgentProtocol.java

```diff
@@ -44,6 +44,15 @@
  * is generated once and used forever, which makes this whole scheme
  * less secure.
  *
+ * <h2>UI Extensions</h2>
```

Comment:
I think you meant to place this Javadoc on `AgentProtocol`, the extension point, not on this implementation.